src/components/ApplicationGroup/Details/TriggerView/BulkCITrigger.tsx

```diff
@@ -173,7 +173,7 @@ export default function BulkCITrigger({
                         branchNames += `${branchNames ? ',' : ''}${material.value}`
                     }
                 }
-                return !branchNames ? null : getCIBlockState(appDetails.ciPipelineId, appDetails.appId, branchNames)
+                return !branchNames || !getCIBlockState? null : getCIBlockState(appDetails.ciPipelineId, appDetails.appId, branchNames)
```

Comment:
Please revert this check. It is not required

Reply:
done